osu.Game/Online/Rooms/Room.cs

```diff
@@ -37,6 +37,17 @@ public class Room : IDeepCloneable<Room>
         [JsonProperty("channel_id")]
         public readonly Bindable<int> ChannelId = new Bindable<int>();
 
+        [JsonProperty("current_playlist_item")]
+        [Cached]
+        public readonly Bindable<PlaylistItem> CurrentPlaylistItem = new Bindable<PlaylistItem>();
+
+        [JsonProperty("playlist_item_stats")]
+        [Cached]
+        public readonly Bindable<RoomPlaylistItemStats> PlaylistItemStats = new Bindable<RoomPlaylistItemStats>();
+
+        [JsonProperty("difficulty_range")]
+        public readonly Bindable<RoomDifficultyRange> DifficultyRange = new Bindable<RoomDifficultyRange>();
```

Comment:
These shouldn't probably be serialised back to web. Other properties use the janky `ShouldSerializeFoo()` implicit methods implicitly used by newtonsoft for this.

Reply:
Won't cause any problems though. I'd actually probably tend towards removing the others instead just because this is hacky and osu-web is responsible for picking out what it needs.

Reply:
I was going to look at making this opt-in, but I guess they *are* required in the other direction so that doesn't make sense. I'll try removing the others and see if everything still works.